### PR TITLE
517 features in embedding widget not working properly

### PR DESCRIPTION
**Pull request by lufre1**
implemented some changes as we discussed for the 2d annotator
@constantinpape

### CODE REVIEW

**Comment by lufre1** [comment]
tested with 2d/tiled_2d annotator, but there seems to be an issue with the 3d automatic segmentation:
```
File ~/Documents/git-repositories/micro-sam/micro_sam/multi_dimensional_segmentation.py:44, in _validate_projection(projection='signle_point')
     42         use_single_point = True
     43     else:
---> 44         raise ValueError(
        projection = 'signle_point'
     45             "Choose projection method from 'mask' / 'points' / 'box' / 'points_and_mask' / 'single_point'. "
     46             f"You have passed the invalid option {projection}."
     47         )
     48 elif isinstance(projection, dict):
     49     assert len(projection.keys()) == 3, "There should be three parameters assigned for the projection method."

ValueError: Choose projection method from 'mask' / 'points' / 'box' / 'points_and_mask' / 'single_point'. You have passed the invalid option signle_point.
```

**Comment by constantinpape** [comment]
I have cleaned up things a bit more and fixed the typo that caused the issue in 3d segmentation that you reported.
We can check  https://github.com/computational-cell-analytics/micro-sam/pull/519#issuecomment-2059355239 tomorrow before merging this.

File: micro_sam/sam_annotator/_widgets.py
Comment (constantinpape) [inline micro_sam/sam_annotator/_widgets.py:599]:
```suggestion
        msg = "No prompts were given. Please provide prompts run interactive segmentation."
```

**Review by constantinpape — APPROVED** [review]
This looks good from my end now. I will merge once the tests pass.

**Pull request merged**
